evalml/tests/model_understanding_tests/test_feature_explanations.py

```diff
@@ -0,0 +1,369 @@
+from unittest.mock import PropertyMock, patch
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from evalml.model_understanding import (
+    get_influential_features,
+    readable_explanation,
+)
+from evalml.pipelines import BinaryClassificationPipeline
+
+elasticnet_component_graph = [
```

Comment:
Let's make this a pytest fixture so it doesn't automatically run if we need this! :)
I'm curious if there are any other pyfixture pipelines we can use, or if this is necessary?

Reply:
Made it into a fixture!

There's a `Logistic Regression Classifier` pyfixture I tried to replace this with, but there's a [woodwork related bug](https://github.com/alteryx/evalml/issues/2863) that prevents the code from running successfully :( so I'll leave this here for now and we can consider swapping the pipeline out once the bug is fixed!